GSASII/GSASIIpwdGUI.py

```diff
@@ -20,6 +20,8 @@
 import random as ran
 import pickle
 import scipy.interpolate as si
+from sympy import symbols, nsimplify, Rational as SRational
```

Comment:
This needs a change to fail gracefully and interface to code that installs packages